src/app/cheat-sheets/game-base/links/links.data.ts

```diff
@@ -245,11 +246,6 @@ export const LINKS_DATA: RawData<LinksData> = {
         text: 'Rain9441',
         url: 'https://pastebin.com/u/rain9441',
       },
-      {
-        text: 'Xterminator',
-        url: 'http://xtvideos.net/factorio/blueprint/',
```

Comment:
I sent xterm a message, waiting on response to verify if he is still hosting his website

Reply:
Looks like his registration is still up to date.

**Registry Expiration:** 2025-05-26 16:58:55 UTC
**Updated:** 2024-07-17 14:24:39 UTC
**Created:** 2017-05-26 16:58:55 UTC

Reply:
I got a response, he doesnt have a site anymore, we can remove, and if he decides to give me bp site link ill add it back in